students/brandon_nguyen/session06/test_mailroom4.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Week6 Excercise mailroom part 4b- the test file
+# Student: Brandon Nguyen - Au2018
+import sys
+import time
+import unittest
+import os
+from decimal import Decimal
+from datetime import datetime, date
+from mailroom4 import sort_sum4_report
+from mailroom4 import update_donation
+from mailroom4 import create_letters_text
+from mailroom4 import create_files
+
+
+input_db = {'FirstNameA LastNameA': [1, 1, 1, 11],
+            'FirstNameB LastNameB': [2, 2, 2, 22],
+            'FirstNameC LastNameC': [3, 3, 3, 33],
+            }
+
+TEST_LETTER = "\n".join(("", "Dear {Name},", "", "Thank you for your "
+                         "kind donation of {LastAmnt:.2f}.",
+                         "It will be put to very good use.", "",
+                         "Sincerely, ", "", "-The Team\n"))
```

Comment:
Isn't this the same as `E_FORMAT` defined in your `mailroom4.py`? If so, it's best to import and reuse it here (`from mailroom4 import E_FORMAT`), rather than duplicating it here.